db/sql/table02_users.sql

```diff
@@ -1,5 +1,6 @@
 CREATE TABLE users (
-    user_id text      NOT NULL UNIQUE PRIMARY KEY CHECK (40 <= LENGTH(user_id) AND LENGTH(user_id) <= 55),
-    created timestamp WITH time zone NOT NULL DEFAULT now()
+    id              serial PRIMARY KEY,
```

Comment:
I pushed up a commit with the change. If you don't like it, I'll revert and fix `users` :)